docs/DeveloperGuide.md

```diff
@@ -238,7 +238,25 @@ _{more aspects and alternatives to be added}_
 
 _{Explain here how the data archiving feature will be implemented}_
 
+### Feature: UpdatedPersonCard for UI
```

Comment:
Just a small comment that we probably need to standardize some way to name the sections under `Implementaion`. I think we can bring this up in the next meeting.